packages/components/src/Drawer/Drawer.scss

```diff
@@ -2,7 +2,7 @@
 
 $tc-drawer-initial-offset: 1000px !default;
 $tc-drawer-bgcolor: #FFF !default;
-$tc-drawer-transition-duration: 0.23s !default;
+$tc-drawer-transition-duration: 230ms !default;
 $tc-drawer-transition-easing: cubic-bezier(0.18, 0.89, 0.32, 1.28) !default;
```

Comment:
to better match ReactCSSTransitionGroup declaration https://github.com/Talend/ui/pull/439/files#diff-6981c5849d690d2f8cd7b6c4952d054bR14